src/Search.js

```diff
@@ -2,8 +2,7 @@ import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
-import makeQueryFunction from '@folio/stripes-components/util/makeQueryFunction';
-import SearchAndSort from '@folio/stripes-smart-components/lib/SearchAndSort';
+import { makeQueryFunction, SearchAndSort } from '@folio/stripes-components';
```

Comment:
Is this supposed to be `@folio/stripes-smart-components`?

Reply:
Yes 😞 Fixed